code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2090,6 +2149,7 @@
 	color = "#AAAAAA55"
 	taste_description = "water"
 	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	pH = 15
```

Comment:
You do realize this makes peacekeeper borgs deal 30 `BRUTE` every time they inject someone with pax right?